resources/puppetlabs/lein-ezbake/template/global/ext/redhat/init.suse.erb

```diff
@@ -43,6 +43,7 @@ config=$CONFIG
 PATH=/sbin:/usr/sbin:/bin:/usr/bin
 JARFILE="<%= EZBake::Config[:uberjar_name] %>"
 JAVA_ARGS="${JAVA_ARGS} -cp ${INSTALL_DIR}/${JARFILE} clojure.main -m <%= EZBake::Config[:main_namespace] %> --config ${CONFIG} -b ${BOOTSTRAP_CONFIG}"
+TK_ARGS="${TK_ARGS}"
```

Comment:
I don't think you need this, I believe this will be defined by the defaults file